cpu/cc26x0/Kconfig

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020 Locha Inc
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config CPU_FAMILY_CC26X0
```

Comment:
I'm a bit confused here about the family and the series having the same name. Would it be OK to define a `CC26xx` series? I know the doc in `cpu/Kconfig` says that the family selects the core, but I think we need to change that, as sometimes it is selected by the series as well.

Reply:
The CPU family in this case would be `CC26X0_CC13X0` which both use a Cortex-M3 (RIOT doesn't define a board for `cc13x0` however, they're compatible).

That would leave us with `CC26x0` series and when wanted a possible `CC13x0` series. Are you ok with it?

Reply:
Yes, this would be similar to the symbols in `cc26x2_cc13x2` right?

Reply:
Yes, same way

Reply:
I've updated the family to `CC26X0_CC13X0`